configuration/byron-mainnet/configuration.yaml

```diff
@@ -0,0 +1,287 @@
+setupBackends:
+  - KatipBK
+
+# This specifies the default backends that trace output is sent to if it
+# is not specifically configured to be sent to other backends.
+defaultBackends:
+  - KatipBK
+
+# EKG is a simple metrics monitoring system. Uncomment the following to enable
+# this backend and listen on the given local port and point your web browser to
+# http://localhost:12788/
+# hasEKG: 12788
+
+# The Prometheus monitoring system exports EKG metrics. Uncomment the following
+# to listen on the given port. Output is provided on
+# http://localhost:12789/metrics
+# hasPrometheus:
+#   - "127.0.0.1"
+#   - 12789
+
+# To enable the legacy 'TraceForwarder' backend, uncomment the following setting. Log
+# items are then forwarded based on an entry in 'mapBackends' to a separate
+# process running a 'TraceAcceptor'.
+# Example using UNIX pipes:
+# traceForwardTo:
+#   tag: RemotePipe
+#   contents: "logs/pipe"
+#
+# Example using Windows named pipes:
+# traceForwardTo:
+#   tag: RemotePipe
+#   contents: "\\\\.\\pipe\\acceptor"
+#
+# Example using network socket:
+# traceForwardTo:
+#   tag: RemoteSocket
+#   contents:
+#     - "127.0.0.1"
+#     - "2997"
+
+# For the Katip logging backend we must set up outputs (called scribes)
+# The available types of scribe are:
+#   FileSK for files
+#   StdoutSK/StderrSK for stdout/stderr
+#   JournalSK for systemd's journal system
+#   DevNullSK ignores all output
+# The scribe output format can be ScText or ScJson. Log rotation settings can
+# be specified in the defaults below or overridden on a per-scribe basis here.
+setupScribes:
+  - scKind: FileSK
+    scName: "logs/mainnet.log"
+    scFormat: ScText
+
+  - scKind: StdoutSK
+    scName: stdout
+    scFormat: ScText
+
+# For the Katip logging backend this specifies the default scribes that trace
+# output is sent to if it is not configured to be sent to other scribes.
+defaultScribes:
+  - - FileSK
+    - "logs/mainnet.log"
+  - - StdoutSK
+    - stdout
+
+# The default file rotation settings for katip scribes, unless overridden
+# in the setupScribes above for specific scribes.
+rotation:
+  rpLogLimitBytes: 5000000
+  rpKeepFilesNum:  3
+  rpMaxAgeHours:   24
+
+
+##### Coarse grained logging control #####
+
+# Trace output from whole subsystems can be enabled/disabled using the following
+# settings. This provides fairly coarse grained control, but it is relatively
+# efficient at filtering out unwanted trace output.
+
+# Trace BlockFetch client.
+TraceBlockFetchClient: False
+
+# Trace BlockFetch decisions made by the BlockFetch client.
+TraceBlockFetchDecisions: False
+
+# Trace BlockFetch protocol messages.
+TraceBlockFetchProtocol: False
+
+# Serialised Trace BlockFetch protocol messages.
+TraceBlockFetchProtocolSerialised: False
+
+# Trace BlockFetch server.
+TraceBlockFetchServer: False
+
+# Trace BlockchainTime.
+TraceBlockchainTime: False
+
+# Verbose tracer of ChainDB
+TraceChainDb: True
```

Comment:
We might want to turn off most of the irrelevant logging, except for `TraceChainDb` and `TraceMempool`, since those are the only ones that are actually useful for debugging transactions and rollbacks.

Reply:
Yes, I was thinking that as well.